PWGHF/HFC/Tasks/taskFlow.cxx

```diff
@@ -271,8 +527,41 @@ struct HfTaskFlow {
     registry.fill(HIST("Data/TpcTpc/HadronHadron/MixedEvent/hNtracksMixing"), Ntracks);
   }
 
+  // ---- MC : TPC-TPC h-h Mixed Event QA ----
+  template <typename TTracks>
+  void fillTpcTpcChChMixedEventQAmc(float multiplicity, float vz, TTracks const& tracks)
+  {
+    if constexpr (std::is_same_v<TracksWDcaSelMC, TTracks>) { // if MC Rec
+      registry.fill(HIST("MC/Rec/TpcTpc/HadronHadron/MixedEvent/hMultiplicityMixing"), multiplicity);
+      registry.fill(HIST("MC/Rec/TpcTpc/HadronHadron/MixedEvent/hVtxZMixing"), vz);
+    } else { // if MC Gen
+      registry.fill(HIST("MC/Gen/TpcTpc/HadronHadron/MixedEvent/hMultiplicityMixing"), multiplicity);
+      registry.fill(HIST("MC/Gen/TpcTpc/HadronHadron/MixedEvent/hVtxZMixing"), vz);
+    }
+
+    int Ntracks = 0;
+    for (const auto& track1 : tracks) {
+      Ntracks++;
```

Comment:
Why can't you use `tracks.size()`?

Reply:
No specific reason to not use it, so I changed the code to use it